CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 See the [releases](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases) for details on bug fixes and added features.
 
+8.0.3
+=====
+### Performance improvements
+- Improves performance during issuer validation by replacing string comparison with span comparison. See PR [#2826](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2826).
+
+### New features
+- Add optional check to prevent using keys that are shared across multiple clouds. See PR [#2804](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2804) for details.
```

Comment:
```suggestion
- Add optional check to prevent using keys that are shared across multiple clouds. See issue [#2832](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2832) for details.
```